blocks/library/text/index.js

```diff
@@ -44,6 +46,11 @@ registerBlockType( 'core/text', {
 					/>
 				</BlockControls>
 			),
+			focus && (
+				<InspectorControls>
+					<Toggle checked={ !! dropCap } onChange={ toggleDropCap } />
```

Comment:
We probably need a label, Should we create an inspector ToggleControl (same as range and text)?